ironic/drivers/modules/drac/management.py

```diff
@@ -183,10 +189,17 @@ def set_boot_device(node, device, persistent=False):
     :raises: DracOperationError on an error from python-dracclient.
     """
 
-    drac_job.validate_job_queue(node)
-
     client = drac_common.get_drac_client(node)
 
+    unfinished_jobs = drac_job.list_unfinished_jobs(node)
```

Comment:
Can you add a comment describing why this change is necessary?

Reply:
Done